main.go

```diff
@@ -1,7 +1,14 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+
+	_ "github.com/iterableio/api/config"
+	"github.com/iterableio/api/db"
+)
 
 func main() {
-	fmt.Printf("Hello World.\n")
+	fmt.Printf("Starting db\n")
```

Comment:
We'll probably need to come up with better logs when you merge your http stuff. For now it's fine.